### DISCUSSION TITLE
Holiday token to NTF fail :(

### DISCUSSION THREAD

**Discussion by xps4less** [Chia-Network/chia-blockchain#17400] [Support]
Hello,
 
I have sent 2 holiday token to the address advised, but never received any nfts back as promised.
It has been over a 2 weeks already.
 
Was sent to xch184hksatuenaz5rddcj3kvdf3mjhs0ud5a47y90340vu8hq739q7sm2uhf6

Please help

**Comment by BrandtH22** [comment]
Hey @xps4less , unfortunately the Holiday token redemption was sunset last year (November 10th, 2023) and we can no longer access the redemption system due to the keys no longer being available. 

We announced the sunsetting of the redemption tool 30 days prior on our various social channels and no longer have the ability to redeem tokens sent to that address. 

I am sorry for any inconvenience and greatly appreciate your understanding 👍

**Comment by xps4less** [comment]
So basically I just wasted 2 tokens right?!

**Reply by BrandtH22** [reply]
@xps4less , unfortunately the redemption was sunset and we no longer have access to the system :/

I am sorry we cannot be of more help and appreciate your understanding, I am going to close this ticket but do not hesitate to reopen it or join our discord server if other questions arise 👍 
https://discord.gg/chia
